plugins/polio/api/vaccines/supply_chain.py

```diff
@@ -22,14 +22,15 @@
 AR_SET = "vaccinearrivalreport_set"
 
 
-def validate_rounds_and_campaign(data, current_user=None):
+def validate_rounds_and_campaign(data, current_user=None, force_rounds=True, force_campaign=True):
```

Comment:
> But I don't understand why rounds and campaing are required at creation time, but optional at update time?

So based on our discussion, I understand that `data.get("rounds")` and `data.get("campaign")` can be None in case of a `PATCH` request.

Suggestions to make it easier to understand:

- add a comment that explain this
- or replace `force_rounds=True` and `force_campaign=True` by a single argument `create=True`